src/app/services/http/gb-backend-http.service.ts

```diff
@@ -40,18 +40,18 @@ export class GbBackendHttpService {
    * Get the queries that the current user has saved.
    * @returns {Observable<Query[]>}
    */
-  getQueries(): Observable<GbBackendQuery[]> {
+  getQueries(): Observable<CohortRepresentation[]> {
```

Comment:
maybe the naming could be getCohorts

Reply:
btw, the name 'CohortRepresentation' sounds odd, not sure how to improve it though.